tests/utils/focus/getActiveElement.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  defineShadowInputCustomElementIfNotDefined,
+  ShadowInput,
+} from '../../_helpers/shadow-input'
+import {setup} from '#testHelpers'
+import {getActiveElementOrBody} from '#src/utils'
+
+test('focused input element', async () => {
+  const {element} = setup('<input />')
+
+  expect(getActiveElementOrBody(document)).toBe(element)
+})
+
+test('focus should be body', async () => {
+  const {element} = setup<HTMLInputElement>('<input />', {focus: false})
+  element.blur()
+  expect(getActiveElementOrBody(document)).toBe(document.body)
+})
+
+describe('on shadow DOM', () => {
+  test('get focused element inside shadow tree', async () => {
+    defineShadowInputCustomElementIfNotDefined()
+    const {element} = setup('<shadow-input></shadow-input>')
+
+    expect(getActiveElementOrBody(document)).toBe(
+      element.shadowRoot?.querySelector('input'),
+    )
+  })
+  test('focus just body', async () => {
+    defineShadowInputCustomElementIfNotDefined()
+    const {element} = setup<ShadowInput>('<shadow-input></shadow-input>', {
+      focus: false,
+    })
+    element.blur()
+    expect(getActiveElementOrBody(document)).toBe(document.body)
+  })
```

Comment:
It still doesn't test anything about `getActiveElementOrBody`. It merely ensures that after `element.blur()` the test setup is the same as in the other test with `{focus: false}` above.

Reply:
What I wanted to test is that nothing is currently focused. I don't see how this is wrong, but I can remove the test if you want.

Reply:
The test suggests that `getActiveElementOrBody` should have a specific behavior on this state of the document. But the created state is exactly the initial state which is already covered by a test above. Therefore it just obfuscates the requirements for `getActiveElementOrBody` instead of describing them.

Reply:
Gotcha. I removed the test.